### ISSUE TITLE
Feature Request: Preserving asset bundle manifest

### ISSUE THREAD

**Issue by rammohan-b** [mitmadness/AssetBundleCompiler#1]
The manifest file Unity generates when creating a asset bundle is now lost when the GeneratedAssetBundles folder is cleaned up when the build completes. This file contains useful information like the CRC of the asset bundle. There could be an option to copy the manifest file too to the destination provided in the to() call.

**Comment by toverux** [comment]
Hello @brammoha, nice feature request :) I will try to allocate a little time to this library again in the next days and will keep you informed.

**Comment by toverux** [comment]
This has landed in [**v1.5.0** (release notes)](https://github.com/mitmadness/AssetBundleCompiler/releases/tag/v1.5.0) (this is also the 100th commit) ! This version also brings a bugfix and (official) Unity 2017 support. 

You can retrieve the manifest in JavaScript now (useful for storing the CRC in a database, logging, etc):

```ts
// The promise gets resolved with the manifest as a plain JS object
const manifest = await bundle('...').to('...');

console.log(`Generated assetbundle with CRC ${manifest.CRC}`)

/* manifest = { 
    CRC: 2924050344,
    Assets: ['Assets/CopiedAssets/MyAsset.jpg'],
    ...etc...
} */
```

Or you can, as originally needed, get the manifest file alongside the asset bundle:

```ts
const manifest = await bundle('...')
    // manifestFile can take a path or a fs.WriteStream too
    .to('/path/to/resources.assetbundle', { manifestFile: '/path/to/resources.assetbundle.manifest' });
```
